loadtest/control/simulcontroller/utils_test.go

```diff
@@ -110,3 +110,40 @@ func TestPickAction(t *testing.T) {
 		require.Greater(t, res[1], res[3])
 	})
 }
+
+func TestSplitName(t *testing.T) {
+	testCases := []struct {
+		input, prefix, typed string
+	}{
+		{
+			input:  "testuser-1",
+			prefix: "testuser-",
+			typed:  "1",
+		},
+		{
+			input:  "testuser999",
+			prefix: "",
+			typed:  "testuser999",
+		},
+		{
+			input:  "téstüser-999",
+			prefix: "",
+			typed:  "téstüser-999",
+		},
+		{
+			input:  "testuser",
+			prefix: "",
+			typed:  "testuser",
+		},
+		{
+			input:  "testuser-100a",
+			prefix: "testuser-",
+			typed:  "100a",
+		},
+	}
+	for _, tc := range testCases {
+		ePrefix, eTyped := splitName(tc.input)
```

Comment:
Not sure what does `e` mean here. Any reason not to keep it `prefix, typed` ?

Reply:
It's a holdover from a previous version of the tests. I'll update.